client.go

```diff
@@ -294,6 +295,22 @@ func (c *Client) SetAuthToken(token string) *Client {
 	return c
 }
 
+// SetAuthScheme method sets the auth token scheme type in the HTTP request. For Example:
```

Comment:
@rigrassm I think its good to include the RFC and IANA links with few texts in the godoc would be helpful for the user.